### PR TITLE
need image tag for heartbeat and listeners

### PR DESCRIPTION
**Pull request by tcnichol**
Without an image tag my local heartbeat and listeners docker images were not being updated, causing issues. 

I found that adding the tags fix the problem, so I added them for these 2 docker images based on what looked like the most current tags. Updated for docker-dev and docker-compose.

### CODE REVIEW

**Review by lmarini — CHANGES_REQUESTED** [review]
Thanks! Let's use `:latest` instead since that is the standard.

**Comment by tcnichol** [comment]
> Thanks! Let's use `:latest` instead since that is the standard.

Fixed.

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -139,7 +139,7 @@ services:
       - elasticsearch:/usr/share/elasticsearch/data
     
   extractors-heartbeat:
-    image: "clowder/clowder2-heartbeat"
+    image: "clowder/clowder2-heartbeat:latest"
```
[the remark this answers is not in the document]
Comment (lmarini) [reply docker-compose.dev.yml:142]: ok, sorry, let's do that then @tcnichol thank you

[the remark this answers is not in the document]
Comment (tcnichol) [reply docker-compose.dev.yml:142]: I will change it back to main.

**Review by ddey2 — APPROVED** [review]

**Review by lmarini — APPROVED** [review]

**Pull request merged**